config/solrconfig.xml

```diff
@@ -72,18 +72,11 @@
        The examples below can be used to load some solr-contribs along
        with their external dependencies.
     -->
-  <lib dir="${solr.install.dir:../../../..}/contrib/extraction/lib" regex=".*\.jar" />
-  <lib dir="${solr.install.dir:../../../..}/dist/" regex="solr-cell-\d.*\.jar" />
 
-  <lib dir="${solr.install.dir:../../../..}/contrib/clustering/lib/" regex=".*\.jar" />
-  <lib dir="${solr.install.dir:../../../..}/dist/" regex="solr-clustering-\d.*\.jar" />
-
-  <lib dir="${solr.install.dir:../../../..}/contrib/langid/lib/" regex=".*\.jar" />
-  <lib dir="${solr.install.dir:../../../..}/dist/" regex="solr-langid-\d.*\.jar" />
-
-  <lib dir="${solr.install.dir:../../../..}/contrib/velocity/lib" regex=".*\.jar" />
```

Comment:
"Velocity" was deprecated and removed from Solr 9.0.  There is apparently a 3rd party plugin that could replace it, but I'm not really sure we were even using it in the first place.  Velocity seems to be a UI interface for making queries (and I wasn't able to get it to pull up on Staging, so it may be that it's been broken for awhile).  From what I can tell, this library is not used as part of the API, just a UI extension.

Reply:
Does the "Solr dashboard" still work? Current production one is at http://search-solr.apps.int.nsidc.org:8983/solr/#/